.changeset/config.json

```diff
@@ -4,6 +4,6 @@
   "commit": false,
   "linked": [],
   "access": "restricted",
-  "baseBranch": "master",
+  "baseBranch": "v0",
```

Comment:
What is baseBranch used for I wonder? I hope we can still release to multiple branches.

Reply:
This file will need to be different in each version branch, but I don't think that's a problem?

Reply:
Here is what that option is for though: https://github.com/atlassian/changesets/blob/0f0b6ad10744de9a8a32305796bd78d33bb1a426/packages/cli/CHANGELOG.md#minor-changes-5